res/layout/game_activity.xml

```diff
@@ -1,21 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Ah, that pertains only to the ControllerProfileActivity.  Sorry to startle you.  It has to do with the way the mapping process is implemented, which imposes special requirements for that particular activity.

I tested on my tablet and didn't experience any issues with the peripheral controller I was using.